hyrax.gemspec

```diff
@@ -26,6 +26,12 @@ EOF
   spec.version       = Hyrax::VERSION
   spec.license       = 'Apache2'
 
+  # Be pessimistic, rails does not follow sem-ver conventions and
+  # minor version releases can include breaking changes; see
+  # http://guides.rubyonrails.org/maintenance_policy.html
+  # https://github.com/projecthydra-labs/hyrax/issues/1028
+  spec.add_dependency 'rails', '~> 5.0'
```

Comment:
It isn't pessimistic now (but I argue that it should be, and lost).  When it's not pessimistic, this comment serves as a reminder and a warning for anyone looking at this line of code.  It's easy to fall into a habit of thinking everything is sem-ver, but rails is not.  I think it's useful to have this reminder right here where it counts.

If y'all want it gone or changed, please indicate "delete line x->y" or "delete all comments" or give me exactly what you want the comment to be?